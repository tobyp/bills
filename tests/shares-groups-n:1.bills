#!./bills
ABC	D.	3.00