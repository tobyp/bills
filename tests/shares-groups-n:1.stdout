todo: D pays 1.0 to A
todo: D pays 1.0 to B
todo: D pays 1.0 to C
